dpnp/dpnp_iface_sorting.py

```diff
@@ -68,18 +70,22 @@ def _wrap_sort_argsort(a, _sorting_fn, axis=-1, kind=None, order=None):
         raise NotImplementedError(
             "kind keyword argument can only be None or 'stable'."
         )
+    if stable is None or not stable:
```

Comment:
In general it looks that we can support `None` and `False` values for `stable` keyword, because dpctl does that.